### PR TITLE
Login styling and profile page

### PR DESCRIPTION
**Pull request by abehara2**

### CODE REVIEW

**Comment by arpanlaha** [comment]
Cat Gif

File: client/pages/register.jsx [hunk 1]
```diff
@@ -167,181 +141,349 @@ export default function RegisterPage(props) {
     const response = await result.json();
     setPinMessage(response.message);
   };
+  const options = [{ value: "student", label: "student" }];
+  const anonOptions = [
+    { value: true, label: "Anonymous Account" },
+    { value: false, label: "Visible Account" }
+  ];
+  const security_questions = [
```
Comment (yousefa00) [inline client/pages/register.jsx:149]: We shouldn't hardcode these, use the provided getSecurityQuestions endpoint

File: client/pages/register.jsx [hunk 2]
```diff
@@ -167,181 +141,349 @@ export default function RegisterPage(props) {
...
+                  <Col xs="9">
+                    <Form>
+                      <FormGroup>
+                        <Input
+                          type="password"
+                          name="password2"
+                          minLength="8"
+                          maxLength="64"
+                          value={password2}
+                          onChange={e => setPassword2(e.target.value)}
```
Comment (yousefa00) [inline client/pages/register.jsx:262]: I think we can just do `{setPassword2}` here. Correct me if I'm wrong though. There's a few places you would need to change if thats the case.

File: client/public/styles/login.scss [hunk 3]
```diff
@@ -0,0 +1,159 @@
+.auth-card-wrapper {
+    display: flex;
+    justify-content: center;
+    
+  }
+  .fontAll {
+    font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
```
Comment (yousefa00) [inline client/public/styles/login.scss:7]: Can we use the font consistent with the rest of the website here?

File: client/public/styles/signupPage.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  font-weight: bolder;
+  font-size: 144px;
+  text-shadow: 4px 4px 2px white;
+}
+.motto{
+  font-size: 3rem;
+  margin-top: 5%;
+  margin-left: 5%;
+  margin-right: 5%;
+  height : 800px;
```
Comment (yousefa00) [inline client/public/styles/signupPage.scss:41]: We want to refrain from using px values wherever possible.

File: client/utils/apiWrapper.js [hunk 5]
```diff
@@ -608,24 +611,25 @@ export const getComments = model_id => {
...
-            "x-access-token": localStorage.getItem("token")
-          }
-        }
-      )
-      .catch(error => ({
-        type: "SAVE_UPVOTE_FAIL",
-        error
-      }));
+  if (!validUser) {
+    return;
```
Comment (yousefa00) [inline client/utils/apiWrapper.js:615]: instead, wrap content in `if (validUser) { ... }` for consistency

**Pull request merged**